<launch>

  <!--include file="$(find summit_xl_apc_moveit_config)/launch/demo.launch"/-->

  <!--node name="dmp_service_node" pkg="dmp_service_simulation" type="dmp_service_simulation_node" respawn="false" output="screen"/-->

  <node name="wam_tf_publisher" pkg="wam_setup_tf" type="wam_tf" respawn="false" output="screen"/>
  
  <!--node name="detect_shelf" pkg="detect_shelf" type="detect_shelf_node" respawn="false" output="screen"/-->

  <node name="lines_rack_det" pkg="lines_rack_det" type="lines_rack_det_node" respawn="false" output="screen"/>

  <node name="wam_ikfast_7dof_service" pkg="wam_ikfast_7dof_service" type="wam_ikfast_7dof_service_node" respawn="false" output="screen"/>

  <node name="kinect_robot_tf_node" pkg="kinect_to_robot_tf" type="kinect_robot_tf" respawn="false" output="screen"/>

  <include file="$(find freenect_launch)/launch/examples/freenect-registered-xyzrgb.launch"/>
 
  <!--node name="publish_pointcloud" pkg="publish_pointcloud" type="publish_pointcloud" respawn="false" output="screen"/-->

  <!--node name="trajectory_service" pkg="trajectory_service" type="trajectory_service" respawn="false" output="screen"/-->

  <!--node name="trajectory_generation_bottom" pkg="trajectory_generation_bottom" type="trajectory_generation_bottom_node" respawn="false" output="screen"/-->

  <!-- node name="rosserial_python" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0" respawn="false" output="screen"/ -->

  <!--node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="camera_frame_id" value="usb_cam" />
  </node-->

  <!--node name="rgb_object_detection" pkg="rgb_object_detection" type="rgb_object_detection_node" respawn="false" output="screen"/-->

  <!--node name="product_count" pkg="product_count" type="product_count" respawn="false" output="screen"/-->

  <!--node name="stow_traj_planning" pkg="stow_traj_planning" type="stow_traj_planning_node" respawn="false" output="screen"/-->

  <!--node name="webcam_record" pkg="webcam_record" type="webcam_record_node" respawn="false" output="screen"/-->

  <!--node name="xwam_motion_node" pkg="xwam_motion" type="xwam_motion_node" respawn="false" output="screen"/-->

  <node name="xwamotion_laser_node" pkg="xwamotion_laser" type="xwamotion_laser_node" respawn="false" output="screen"/>

  <node name="trajectory_rpt_side" pkg="trajectory_rpt" type="trajectory_rpt_side" respawn="false" output="screen"/>

  <!--node name="trajectory_stow" pkg="trajectory_rpt" type="trajectory_stow" respawn="false" output="screen"/-->

  <!--node name="trajectory_rpt_node" pkg="trajectory_rpt" type="trajectory_rpt_node" respawn="false" output="screen"/-->

</launch>

